test/ObjectField_test.js

```diff
@@ -50,7 +50,10 @@ describe("ObjectField", () => {
     });
 
     it("should render a customized title", () => {
-      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+      const CustomTitleField = ({title, required}) => {
+        const legend = required ? title + '*' : title;
+        return <div id="custom">{legend}</div>;
+      };
```

Comment:
I can do that, however, isn't CustomTitleField something that is supposed to be overridden? Meaning, the implementation and handling of the required field is the responsibility of whomever implements CustomTitleField. Or is your idea that we should test my reference implementation of the CustomTitleField?